vpdq/setup.py

```diff
@@ -1,40 +1,32 @@
 from setuptools import setup
 from setuptools.extension import Extension
-from setuptools.command.build_py import build_py
+from setuptools.command.build_ext import build_ext                                     
 import sys
 import subprocess
 import os
 import numpy
 from pathlib import Path
 
-read_me = Path.cwd() / Path("README.md")
+DIR = Path(__file__).parent
+read_me = DIR / Path("vpdq/README.md")
 long_description = read_me.read_text()
+version = (DIR / "vpdq/version.txt").read_text(encoding="utf-8").strip()
 
-
-class Build(build_py):
-    """Customized setuptools build command - builds protos on build."""
-
+class build_ext(build_ext):
     def run(self):
-        os.chdir("cpp")
-        command = ["mkdir", "build"]
-        subprocess.call(command)
-        os.chdir("build")
-        command = ["cmake", ".."]
-        if subprocess.call(command) != 0:
-            sys.exit(-1)
+        os.chdir("vpdq/cpp")
```

Comment:
blocking: I think this will have unexpected results unless you are in a specific directory.

I generally recommend against using chdir, and instead use known paths for everything based on the relative path of the current file, or paths relative to modules in the directory.

subprocess.call() and it's friends all take a "cwd" argument that will set the current working directory, so there is no need to change the global working directory of the environment.